dGame/dBehaviors/SwitchBehavior.cpp

```diff
@@ -30,7 +30,7 @@ void SwitchBehavior::Handle(BehaviorContext* context, RakNet::BitStream* bitStre
 
 	Game::logger->LogDebug("SwitchBehavior", "[%i] State: (%d), imagination: (%i) / (%f)", entity->GetLOT(), state, destroyableComponent->GetImagination(), destroyableComponent->GetMaxImagination());
 
-	if (state || (entity->GetLOT() == 8092 && destroyableComponent->GetImagination() >= m_imagination)) {
```

Comment:
What was this edge case for?

Reply:
Likely server side verification of the skill, but as we learned with some earlier PRs we shouldn't be trying to do server side calculation of skill handles as it may cause desyncs.  At least thats my thought process when it comes to skill Handles now with what we know

Reply:
I did test that racing still works as expected